src/style.css

```diff
@@ -1 +1,76 @@
 /* Styles go here */
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.header {
+  width: 1200px;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  box-sizing: border-box;
+  margin: 0 auto;
+  padding: 0 50px;
+  background-color: #fff;
+}
+
+@media (max-width: 1024px) {
+  .header {
+    width: 1024px;
+  }
+}
+
+.header__logo {
+  display: inline-block;
+  width: 40px;
+  height: 40px;
+  padding: 10px 0;
+}
+
+.menu {
+  display: flex;
+  padding: 0;
+  margin: 0;
```

Comment:
you can reset these spaces on the top file using ul tag. In the future, you will do this on the reset.css file